releases/cosmwasm/2025-01-ITS-v1.1.0.md

```diff
@@ -27,8 +27,15 @@ Changes in this release:
 
 Upload new ITS Hub contract
 
+| environment | INIT_ADDRESSES    |  RUN_AS_ACCOUNT |
+| :-----: | :---: | :---: |
+| devnet-amplifier | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9  | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
+| stagenet | axelar1pumrull7z8y5kc9q4azfrmcaxd8w0779kg6anm,axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar12qvsvse32cjyw60ztysd3v655aj5urqeup82ky    | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
+| testnet | axelar1uk66drc8t9hwnddnejjp92t22plup0xd036uc2,axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar12f2qn005d4vl03ssjq07quz6cja72w5ukuchv7   | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
+| mainnet | axelar1uk66drc8t9hwnddnejjp92t22plup0xd036uc2,axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj,axelar1nctnr9x0qexemeld5w7w752rmqdsqqv92dw9am   | axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj   |
+
 ```bash
-node cosmwasm/submit-proposal.js store -c InterchainTokenService -t "Upload InterchainTokenService contract v1.1.0" -d "Upload InterchainTokenService contract v1.1.0" -r $RUN_AS_ACCOUNT --deposit 2000000000 -a ./wasm
+node cosmwasm/submit-proposal.js store -c InterchainTokenService -t "Upload InterchainTokenService contract v1.1.0" -d "Upload InterchainTokenService contract v1.1.0" -r $RUN_AS_ACCOUNT --deposit 2000000000 --instantiateAddresses $INIT_ADDRESSES -a ./wasm
```

Comment:
the -a ./wasm seems a bit error prone to me. We should tack on to these deployments, command to download the contract and place it in a version tagged directory and then point to that directory.

Alternatively, you can add a pre-run command that matches the checksum of the contract against the release on github or in cloudflare.

Reply:
just don't want to run into a situation where we upload the contract on mainnet for instance, only to realize it was the wrong one, and we wasted 3 days.

Reply:
Looks like the easier of 2 is document cmds to download bytecode and checksum matching here.

Reply:
Added step to download wasm bytecode and verify checksum.